docs/administrator-guide/configuration/storage/README.md

```diff
@@ -4,18 +4,18 @@ order: 2
 
 # Storage configuration
 
-Kestra needs [internal storage](../../../architecture#storage) in order to store file proceeded (uploaded files & between tasks).
+Kestra needs an [internal storage](../../../architecture#storage) in order to store data processed by flow tasks (files from flow inputs and data stored as task outputs).
 
-The default storage is the Local storage (**not production ready**).
+The default internal storage is the local storage which is **not suitable for production** as it will store data inside a local folder on the host filesystem..
```

Comment:
```suggestion
The default internal storage is the local storage which is **not suitable for production** as it will store data inside a local folder on the host filesystem.
```